<?xml version="1.0" encoding="UTF-8"?>
<?xml-stylesheet type="text/xsl" href="ivy-report.xsl"?>
<ivy-report version="1.0">
	<info
		organisation="default"
		module="lab-5_2.12"
		revision="0.1"
		conf="test-internal"
		confs="compile, runtime, test, provided, optional, compile-internal, runtime-internal, test-internal, plugin, pom, scala-tool"
		date="20180421080824"/>
	<dependencies>
		<module organisation="org.scala-lang" name="scala-library">
			<revision name="2.12.5" status="release" pubdate="20180316154746" resolver="sbt-chain" artresolver="sbt-chain" homepage="http://www.scala-lang.org/" extra-info.apiURL="http://www.scala-lang.org/api/2.12.5/" downloaded="false" searched="false" default="false" conf="default, compile, runtime, default(compile), master" position="0">
				<license name="BSD 3-Clause" url="http://www.scala-lang.org/license.html"/>
				<metadata-artifact status="no" details="" size="2685" time="0" location="C:\Users\Анастасия\.ivy2\cache\org.scala-lang\scala-library\ivy-2.12.5.xml" searched="false" origin-is-local="false" origin-location="https://repo1.maven.org/maven2/org/scala-lang/scala-library/2.12.5/scala-library-2.12.5.pom"/>
				<caller organisation="default" name="lab-5_2.12" conf="test, compile, provided, test-internal, runtime, optional" rev="2.12.5" rev-constraint-default="2.12.5" rev-constraint-dynamic="2.12.5" callerrev="0.1"/>
				<artifacts>
					<artifact name="scala-library" type="jar" ext="jar" status="no" details="" size="5263467" time="0" location="C:\Users\Анастасия\.ivy2\cache\org.scala-lang\scala-library\jars\scala-library-2.12.5.jar">
						<origin-location is-local="false" location="https://repo1.maven.org/maven2/org/scala-lang/scala-library/2.12.5/scala-library-2.12.5.jar"/>
					</artifact>
				</artifacts>
			</revision>
		</module>
		<module organisation="org.scalatest" name="scalatest_2.12">
			<revision name="3.0.5" error="public: unable to get resource for org/scalatest#scalatest_2.12;3.0.5: res=https://repo1.maven.org/maven2/org/scalatest/scalatest_2.12/3.0.5/scalatest_2.12-3.0.5.pom: java.net.UnknownHostException: repo1.maven.org" downloaded="false" searched="false" conf="default(compile)" position="-1">
				<caller organisation="default" name="lab-5_2.12" conf="test, compile, provided, test-internal, runtime, optional" rev="3.0.5" rev-constraint-default="3.0.5" rev-constraint-dynamic="3.0.5" callerrev="0.1"/>
				<artifacts>
				</artifacts>
			</revision>
		</module>
		<module organisation="org.scalctic" name="scalactic_2.12">
			<revision name="3.0.5" error="public: unable to get resource for org/scalctic#scalactic_2.12;3.0.5: res=https://repo1.maven.org/maven2/org/scalctic/scalactic_2.12/3.0.5/scalactic_2.12-3.0.5.pom: java.net.UnknownHostException: Р­С‚РѕС‚ С…РѕСЃС‚ РЅРµРёР·РІРµСЃС‚РµРЅ (repo1.maven.org)" downloaded="false" searched="false" conf="default(compile)" position="-1">
				<caller organisation="default" name="lab-5_2.12" conf="test, compile, provided, test-internal, runtime, optional" rev="3.0.5" rev-constraint-default="3.0.5" rev-constraint-dynamic="3.0.5" callerrev="0.1"/>
				<artifacts>
				</artifacts>
			</revision>
		</module>
	</dependencies>
</ivy-report>
